### ISSUE TITLE
project pages violates CORS and cannot be loaded

### ISSUE THREAD

**Issue by TomasTomecek** [packit/dashboard#114]
We got a report from @msehnout that they cannot load https://dashboard.packit.dev/projects/github.com/osbuild/osbuild

Easily reproducible, the error is:
```
Unable to connect
There was an error retrieving data.
```

Console seems to suggest that:

```
Access to fetch at 'https://prod.packit.dev/api/projects/github.com/osbuild/osbuild' from origin 'https://dashboard.packit.dev' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

Do we just need to set some CORS-related header to make browsers happy?

**Comment by mfocko** [comment]
`https://prod.packit.dev/api/projects/github.com/osbuild`:
```json
[
    {
        "namespace": "osbuild",
        "repo_name": "osbuild",
        "project_url": "https://github.com/osbuild/osbuild/osbuild",
        "prs_handled": 0,
        "branches_handled": 0,
        "releases_handled": 0,
        "issues_handled": 0
    },
    {
        "namespace": "osbuild",
        "repo_name": "osbuild",
        "project_url": "https://github.com/osbuild/osbuild",
        "prs_handled": 28,
        "branches_handled": 4,
        "releases_handled": 0,
        "issues_handled": 2
    }
]
```

related to packit/packit-service#797

**Comment by lachmanfrantisek** [comment]
@mfocko Good catch!

**Comment by mfocko** [comment]
@lachmanfrantisek I could probably fit it in the card, also I'm wondering if it makes sense to have projects with no associated builds, maybe I should check against the DB how many rows does it affect

**Comment by lachmanfrantisek** [comment]
@mfocko Can you please update the card/description to match the actual problem? (So do some database clean up.) What about closing in favour of packit/packit-service#797 update that one to cover this problem?

**Comment by mfocko** [comment]
@lachmanfrantisek will do
